pypeit/pypeit.py

```diff
@@ -469,22 +471,22 @@ def reduce_exposure(self, frames, bg_frames=None, std_outfile=None):
         has_bg = True if bg_frames is not None and len(bg_frames) > 0 else False
         # Is this an IR reduction?
         # TODO: Why specific to IR?
-        # JFH This is not specific to IR, but to b/g subtraction with frames. The flag though is self.ir_redux. Perhaps
+        # JFH This is not specific to IR, but to b/g subtraction with frames. The flag though is self.bkg_redux. Perhaps
         # we should rename this to bg_redux or something like that, since it need not be IR.
         if has_bg:
-            self.ir_redux = True
+            self.bkg_redux = True
```

Comment:
Should we adopt this change from `ir_redux` to `bkg_redux` everywhere?

Reply:
Changed everywhere